torchao/quantization/quantize_/workflows/float8/float8_tensor.py

```diff
@@ -549,19 +549,16 @@ def _quantize_and_scaled_conv3d(
     input_qdata = input_qdata.contiguous(memory_format=torch.channels_last_3d)
     weight_qdata = weight_qdata.contiguous(memory_format=torch.channels_last_3d)
 
-    # move C_in to last dim
-    # after permute: (N, D, H, W, C_in)
-    input_qdata = input_qdata.permute([0, 2, 3, 4, 1])
-
-    # move C_in to last dim
-    # after permute: (C_out, K1, K2, K3, C_in)
-    weight_qdata = weight_qdata.permute([0, 2, 3, 4, 1])
-
     input_scale = input_tensor.scale
     weight_scale = weight_tensor.scale
 
     import mslk.conv  # noqa: F401
```

Comment:
move to top of file?

Reply:
OK, sg. this is not always available, so will need to guard it with `_is_mslk_available`

also currently fbgemm is imported in the is available function: https://github.com/pytorch/ao/blob/428bbcf8e8ec3bdfb0c3ae38e787827d1b540e29/torchao/utils.py#L1141 we'll have to refactor that as well I think